Sources/MUXSDKStatsObjc/MUXSDKPlayerBindingManager.m

```diff
@@ -29,15 +29,43 @@ @interface MUXSDKPlayerBindingManager()
 @end
 @implementation MUXSDKPlayerBindingManager
 
-- (id)init {
+- (nonnull instancetype)init {
     self = [super init];
     if (self) {
-        self.playerReadyBindings = [[NSMutableSet alloc] init];
+        _playerReadyBindings = [[NSMutableSet alloc] init];
+        _customerPlayerDataStore = [[MUXSDKCustomerPlayerDataStore alloc] init];
+        _customerVideoDataStore = [[MUXSDKCustomerVideoDataStore alloc] init];
+        _customerViewDataStore = [[MUXSDKCustomerViewDataStore alloc] init];
+        _customerCustomDataStore = [[MUXSDKCustomerCustomDataStore alloc] init];
+        _playerBindings = [[NSMutableDictionary alloc] init];
     }
     return self;
 }
 
-- (void) onPlayerDestroyed:(NSString *_Nonnull) name {
+- (void)setCustomerData:(nonnull MUXSDKCustomerData *)customerData
+          forPlayerName:(nonnull NSString *)name {
+    MUXSDKCustomerPlayerData *playerData = customerData.customerPlayerData;
+    MUXSDKCustomerVideoData *videoData = customerData.customerVideoData;
+    MUXSDKCustomerViewData *viewData = customerData.customerViewData;
+    MUXSDKCustomData *customData = customerData.customData;
+
+    [_customerPlayerDataStore setPlayerData:playerData
```

Comment:
Prior implementation didn't check `playerData` for nil, kept the same distinction here.